content/storing-in-cloud-must-try.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Storing in cloud, must try
+desc: Why i store my docs in cloud
+author: Raymond WP
+github: raywp
+twitter:
+telegram:
+date: 2022-03-10
+cover: https://www.codebase.co.in/wp-content/uploads/2020/02/Choose-Best-Cloud-Storage-Provider-For-Your-Requirement.jpg
+categories:
+  - cloud
+  - storage
+---
+
+## Which cloud?
+
+Any cloud storage as long as it's not the cloud that can cause rain. Goole Drive, OneDrive, DropBox, etc you name it.
+
+## Origin
+
+Once upon a time, when a friend of mine told me "Cloud is new best friend". So i was thinking to give it a try. Back then i thought by separating disk is enough for data backup-ing (Like C: and D:); but in fact, your local disk could be corrupted, your laptop could be stolen, your data could lost. Since at the time i am writing this, I was a final-year student that love my Final Year Project so much, i can't afford to lose my data, **_I need backup_**.
+
+## What did i do?
+
+I connected my Office Apps (Word, Excel, PowerPoint, etc.) to OneDrive
+[How to add OneDrive to OfficeApps](https://support.microsoft.com/en-us/office/how-to-add-onedrive-as-a-service-3ae78a39-c0de-4c86-8abc-ad519b6de44e)
+[How to add OneDrive as additional storage](https://support.microsoft.com/en-us/office/sync-files-with-onedrive-files-on-demand-1ab17c60-af03-4c63-a24c-bb1e131f76c9)
```

Comment:
Because this is a markdown format, these 3 lines will eventually be one line. If you want a line break, you should have a new line containing nothing. So it's:

```md
Text here

Now we got a new line
```